modules/apps/user/user-test/src/testIntegration/java/com/liferay/user/service/test/UserServiceWhenAddingUserWithDefaultRolesTest.java

```diff
@@ -45,6 +57,62 @@ public class UserServiceWhenAddingUserWithDefaultRolesTest {
 	public static final AggregateTestRule aggregateTestRule =
 		new LiferayIntegrationTestRule();
 
+	@Test
+	public void testMultipleDefaultAssociations() throws Exception {
+		Company company = _companyLocalService.getCompany(
+			TestPropsValues.getCompanyId());
+
+		PortalPreferences portalPreferences =
+			PortletPreferencesFactoryUtil.getPortalPreferences(
+				company.getUserId(), true);
+
+		Group group = GroupTestUtil.addGroup();
+
+		portalPreferences.setValue(
+			"", PropsKeys.ADMIN_DEFAULT_GROUP_NAMES,
+			group.getDescriptiveName());
+
+		UserGroup userGroup = UserGroupTestUtil.addUserGroup();
+
+		portalPreferences.setValue(
+			"", PropsKeys.ADMIN_DEFAULT_USER_GROUP_NAMES, userGroup.getName());
+
+		portalPreferences.setValue(
+			"", PropsKeys.ADMIN_SYNC_DEFAULT_ASSOCIATIONS, "true");
+
+		PortalPreferencesLocalServiceUtil.updatePreferences(
+			company.getCompanyId(), PortletKeys.PREFS_OWNER_TYPE_COMPANY,
+			PortletPreferencesFactoryUtil.toXML(portalPreferences));
+
+		_user = UserTestUtil.addUser(company);
+
+		_groupLocalService.deleteUserGroup(_user.getUserId(), group);
+
```

Comment:
please verify if you can remove this empty line or the sf complains

Reply:
SF complained about it unfortunately